doc/userguide/rules/dcerpc-keywords.rst

```diff
@@ -0,0 +1,56 @@
+DCERPC Keywords
+================
+
+Following keywords can be used for matching on fields in headers and payloads
+of DCERPC packets over UDP, TCP and SMB.
+
+dcerpc.iface
+-------------
+
+Match on the value of the interface UUID in a DCERPC header.
+
+The format of the keyword::
+
+  dcerpc.iface:<uuid>;
```

Comment:
If you look at `rust/src/dcerpc/detect.rs` you will see that this supports more complex options as well.